### PR TITLE
Add Flash messages

### PR DESCRIPTION
**Pull request by rorpyll**

### CODE REVIEW

File: app/controllers/attendences_controller.rb [hunk 1]
```diff
@@ -32,6 +32,6 @@ def update_attendence
 
   def set_attendance
     @attendence = current_employee.attendences.find_by_id params[:id]
-    redirect_to root_path, { notice: 'NOT FOUND :)' } unless @attendence.present?
+    redirect_to root_path, { alert: 'NOT FOUND :)' } unless @attendence.present?
```
Comment (rorshaan) [inline app/controllers/attendences_controller.rb:35]: use I18n

File: app/controllers/daily_tasks_controller.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def show
     else
       @daily_task = current_employee.daily_tasks.find_by_id params[:id]
 
-      redirect_to root_path, { notice: 'You Have Not Access :)' } unless @daily_task.present?
+      redirect_to root_path, { alert: 'You Have Not Access :)' } unless @daily_task.present?
```
Comment (rorshaan) [inline app/controllers/daily_tasks_controller.rb:20]: Use I18n

File: app/controllers/daily_tasks_controller.rb [hunk 3]
```diff
@@ -61,7 +61,7 @@ def update
 
   def set_daily_task
     @daily_task =  DailyTask.find_by_id params[:id]
-    render json: { message: 'Not Found' }, status: 404 unless @daily_task.present?
+    render json: { alert: 'Not Found' }, status: 404 unless @daily_task.present?
```
Comment (rorshaan) [inline app/controllers/daily_tasks_controller.rb:64]: Use I18n

**Review by rorshaan — CHANGES_REQUESTED** [review]

File: app/controllers/attendences_controller.rb (hunk 1 above)
Comment (rorpyll) [reply app/controllers/attendences_controller.rb]: Done

File: app/controllers/daily_tasks_controller.rb (hunk 2 above)
Comment (rorpyll) [reply app/controllers/daily_tasks_controller.rb]: done

File: app/controllers/daily_tasks_controller.rb (hunk 3 above)
Comment (rorpyll) [reply app/controllers/daily_tasks_controller.rb:64]: render json not sport  i18n

Comment (rorpyll) [reply app/controllers/daily_tasks_controller.rb]: done with redirect_to

**Review by rorshaan — APPROVED** [review]

**Pull request merged**